ban/commands/export.py

```diff
@@ -12,10 +27,29 @@ def resources(path, **kwargs):
 
     path    path of file where to write resources
     """
-    resources = [models.PostCode, models.Municipality, models.Group,
-                 models.HouseNumber]
-    with Path(path).open(mode='w', encoding='utf-8') as f:
-        for resource in resources:
-            for data in resource.select().serialize({'*': {}}):
-                f.write(dumps(data) + '\n')
-                reporter.notice(resource.__name__, data)
+    resources = [Municipality, PostCode, Group, HouseNumber]
+    for resource in resources:
+        query = QUERIES.get(resource)
+        filename = '{}.ndjson'.format(resource.__name__.lower())
+        with Path(path).joinpath(filename).open(mode='w') as f:
+            print('Exporting to', f.name)
+            query = query.order_by(resource.pk)
+            results = []
+            for result in helpers.batch(process_resource, query,
+                                        chunksize=1000, total=query.count()):
```

Comment:
I was wondering if it affects performances and if you have to modify these based on your “hardware” config?